src/style.css

```diff
@@ -1 +1,30 @@
-/* add styles here */
+.page__budy {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star:not(:first-child) {
+  margin-left: 4px;
```

Comment:
Be consistent with your margins. It's better to add only top or bottom margin, but not both, to avoid potential margin collapse.